.travis.yml

```diff
@@ -43,6 +43,9 @@ jobs:
     - REPO=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
     - TRAVIS_PYTHON_VERSION=3.6
     - OS=osx-64
+env:
+    global:
+      - MPLBACKEND=agg
```

Comment:
This is where we make sure unit tests are not blocked by interactive figures, right ?

Reply:
Yes, this sets matplotlib to the `agg` backend so not interactive